resources/calibs.sql

```diff
@@ -0,0 +1,2365 @@
+PRAGMA foreign_keys=OFF;
```

Comment:
Is this a stopgap before we have some calibration-ingestion utilities?

Reply:
Yes, exactly, we can toss it away with the associated script when we are ready.